sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -36,6 +36,18 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
   /**
    * For each node, extract properties in the form of a list ['key1', 'key2', 'key3', 'value']
    * into a pair (key1.key2.key3, value).
+   *
```

Comment:
@andrewor14 I will change `For each node, extract properties in the form of a list ['key1', 'key2', 'key3', 'value'] into a pair (key1.key2.key3, value).` to `For each node, extract properties in the form of a list ['key_part1', 'key_part2', 'key_part3', 'value'] into a pair (key_part1.key_part2.key_part3, value).`. Hopefully, this will make the comment easier to understand.